influence_simulator/simulators/indepndent_cascade_simulator.py

```diff
@@ -0,0 +1,48 @@
+from lightning_diffusion.models import IndependentCascadeModel
+from networkx import Graph
+
+from .influence_simulator import InfluenceSimulator, Peak, SimulationResult
+
+
+class IndependentCascadeSimulator(InfluenceSimulator):
+    def __init__(
+        self,
+        graph: Graph,
+        infection_probability: float,
+        *,
```

Comment:
Wasn't it better to force "unexpected" arguments to be keyword-based instead of positional? (i.e. to replace `*` with `**`)?

Reply:
I'm not really sure what you mean. This syntax is there to force the `random_state` argument to be keyword-only (as it is rather _technical_).

Reply:
Ok, I didn't catch that